features/upgrade/node/node-upgrade.feature

```diff
@@ -77,6 +77,6 @@ Feature: Node components upgrade tests
       | cat /etc/kubernetes/kubelet.conf |
     Then the step should succeed
     And the output should contain:
-      | "imageMinimumGCAge": "5m0s"       |
```

Comment:
Instead of hard code to json/yaml format, we can use `the output should match`, then we can regexp in the outputs.